package.json

```diff
@@ -1751,21 +1751,20 @@
             "description": "%configuration.razor.languageServer.debug%",
             "order": 90
           },
-          "razor.trace": {
+          "razor.server.trace": {
+            "scope": "window",
             "type": "string",
-            "default": "Off",
             "enum": [
-              "Off",
-              "Messages",
-              "Verbose"
-            ],
-            "enumDescriptions": [
-              "%configuration.razor.trace.off%",
-              "%configuration.razor.trace.messages%",
-              "%configuration.razor.trace.verbose%"
+              "Trace",
+              "Debug",
+              "Information",
+              "Warning",
+              "Error",
+              "Critical",
+              "None"
             ],
-            "description": "%configuration.razor.trace%",
-            "order": 90
```

Comment:
not sure what this "order" was needed for.

Reply:
Me neither

Reply:
this ensures the Razor settings are grouped and ordered together (cc @Cosifne )